modules/ROOT/pages/endpoint-as2-send.adoc

```diff
@@ -8,16 +8,24 @@ Create the AS2 identifiers for your organization and your trading partners in th
 |===
 |Setting |Description |Required
 
-| Owner
-| The trading partner or host that owns the endpoint.
-| Yes
+|Owner
+|The trading partner or host that owns the endpoint.
+|Yes
 
-| Protocol
-| The protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window.
-| Yes
+|Usage
+|The usage of the endpoint. For receive endpoints, the value in this field is `Send to partners`. The value in this field is automatically generated based on the type of endpoint created.
+|No
+
+|Protocol
+|The protocol selected in the *Protocol* field of the *New endpoint: Receive from partners* window.
+|Yes
+
+|Partner
+|The partner to whom the transaction is sent.
+|Yes
 
 |Endpoint Name
-| The name used to identify the endpoint
+|The name used to identify the endpoint
```

Comment:
In peer elements that use a mixture of fragments and sentences, all elements should end with a period.
When possible, don't mix:
```suggestion
|The name used to identify the endpoint.
```